src/types/types.ts

```diff
@@ -73,3 +73,7 @@ export type VaultWithdrawRequest = {
   vaultAddress: `0x${string}`;
   tokenAmount: bigint;
 };
+
+export type UpdateCountryProps = {
```

Comment:
Where is it used?

Reply:
> Where is it used?

https://github.com/web3-nomad/accelerator-defi-EIP-ui/blob/23e1384e8413c9d2b9a2d96460cfa7e193528318/src/views/manage-identities/ManageIdentities.tsx#L72

This line was hidden from you.